pkg/kubelet/kubelet.go

```diff
@@ -2014,6 +2014,13 @@ func (kl *Kubelet) SyncTerminatingPod(_ context.Context, pod *v1.Pod, podStatus
 		}
 	}
 
+	// compute and update the status in cache once the pods are no longer running
+	apiPodStatus = kl.generateAPIPodStatus(pod, podStatus, true)
+	if podStatusFn != nil {
+		podStatusFn(&apiPodStatus)
```

Comment:
This should not be called here because you already called it once up above

Reply:
Done